# Format comments

## Rule id {#rule-id}

format-comment

## Severity {#severity}

Style

## Description {#description}

Prefer format comments like sentences.

### Example {#example}

Bad:

```dart
// prefer format comments like sentences // LINT
```

Good:

```dart
// Prefer format comments like sentences. 
```
